src/Framework/QueryBuilder/Concerns/MetaQuery.php

```diff
@@ -119,19 +118,20 @@ public function attachMeta($table, $foreignKey, $primaryKey, ...$columns)
             }
 
             $this->join(
-                function (JoinQueryBuilder $builder) use ($table, $foreignKey, $primaryKey, $tableAlias, $column, $metaTable) {
+                function (JoinQueryBuilder $builder) use (
+                    $table,
+                    $foreignKey,
+                    $primaryKey,
+                    $tableAlias,
+                    $column,
+                    $metaTable
+                ) {
                     $builder
                         ->leftJoin($table, $tableAlias)
                         ->on($foreignKey, "{$tableAlias}.{$primaryKey}")
                         ->andOn("{$tableAlias}.{$metaTable->keyColumnName}", $column, true);
                 }
             );
-
-            // If this is non-aggregate column - we have to include it in the GROUP BY statement
-            // otherwise the query will fail on servers that have sql_mode flag set to only_full_group_by
-            if ($groupConcat && !$concat) {
-                $this->groupBy($columnAlias ?: $column);
```

Comment:
@jonwaldstein How does this come into play? My understanding is this only runs into problems when attempting to use `HAVING` or `ORDER BY` on a non-grouped column when `GROUP BY` is present.

Reply:
Have you tried enabling and disabling that `only_full_group_by` flag?

Reply:
@JasonTheAdams `only_full_group_by` is enabled by default on 5.7 so that's what i've been using. Adding an `orderBy` command using a meta value works as expected.  But I just discovered something new...

On develop (PRE-OPTIMIZED query), if all I do is add an ORDER BY command to the donor query `->orderBy('createdAt', 'DESC');` It does _not_ run into any buffer issues. 🤯 

That being said, the POST-OPTIMIZED query in this PR works with or without order by & with `only_full_group_by` enabled.

Reply:
I guess that makes sense in that ordering gives the optimizer a specific direction to work towards, and may even mean that it uses the index to order it.

In any case, I'm glad that we've tested this out. I know it sounds weird, but have you tested disabling it as well? I just want to be 1000% confident we're not shifting problems around on this one. Too much at stake. 😬

Reply:
@JasonTheAdams yes, i've tested both enabled and disabled on MySQL 5.7 & MySQL 8.